etc/profile-m-z/strawberry.profile

```diff
@@ -0,0 +1,33 @@
+# Firejail profile for strawberry
+# Description: A music player and music collection organizer
+# This file is overwritten after every install/update
+# Persistent local customizations
+include strawberry.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/strawberry
+noblacklist ${HOME}/.config/strawberry
+noblacklist ${MUSIC}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+include whitelist-var-common.inc
```

Comment:
`include whitelist-runuser-common.inc`
`include whitelist-usr-share-common` + `whitelist /usr/share/…`

Reply:
I don't know whether strawberry needs anything from `/usr/share` or not, therefore I'm not sure add this or not.

Reply:
```
openat(AT_FDCWD, "/usr/share/X11/locale/locale.alias", O_RDONLY) = 53
openat(AT_FDCWD, "/usr/share/X11/locale/locale.alias", O_RDONLY) = 53
openat(AT_FDCWD, "/usr/share/X11/locale/compose.dir", O_RDONLY) = 53
openat(AT_FDCWD, "/usr/share/X11/locale/en_US.UTF-8/Compose", O_RDONLY) = 53
openat(AT_FDCWD, "/usr/share/icons/Adwaita/cursors/split_h", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/icons/Adwaita/index.theme", O_RDONLY) = 55
openat(AT_FDCWD, "/usr/share/pixmaps/Adwaita/cursors/split_h", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/pixmaps/Adwaita/index.theme", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/icons/default/cursors/split_h", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/icons/default/index.theme", O_RDONLY) = 54
openat(AT_FDCWD, "/usr/share/pixmaps/default/cursors/split_h", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/icons/Adwaita/cursors/split_h", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/icons/Adwaita/index.theme", O_RDONLY) = 54
openat(AT_FDCWD, "/usr/share/pixmaps/Adwaita/cursors/split_h", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/pixmaps/Adwaita/index.theme", O_RDONLY) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/usr/share/icons/Adwaita/cursors/col-resize", O_RDONLY) = 55
```
`include whitelist-usr-share-common` added

Reply:
wusc (whitelist-usr-share-common) includes all the commonly used files, usually you can just search for a strawberry folder in /usr/share.

You package-manager can also help you (https://wiki.archlinux.org/index.php/Pacman_Rosetta#Querying_specific_packages) or `firejail --build`.